ros_android compile issue, on dashing branch with colcon build. Running UBUNTU 18.04 and ROS Dashing

Finished <<< rosidl_typesupport_cpp [0.39s]                                                                               
--- stderr: rosidl_generator_java                                
/opt/ndk/android-ndk-r21d/toolchains/llvm/prebuilt/linux-x86_64/lib/gcc/arm-linux-androideabi/4.9.x/../../../../arm-linux-androideabi/bin/ld: error: cannot find -lrosidl_generator_java__rosidl_typesupport_opensplice_c
clang++: error: linker command failed with exit code 1 (use -v to see invocation)
make[2]: *** [rosidl_generator_java/rosidl_generator_java/msg/librosidl_generator_java_msg__nested__jni__rosidl_typesupport_opensplice_c.so] Error 1
make[1]: *** [CMakeFiles/msgNestedopensplice_c.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
/opt/ndk/android-ndk-r21d/toolchains/llvm/prebuilt/linux-x86_64/lib/gcc/arm-linux-androideabi/4.9.x/../../../../arm-linux-androideabi/bin/ld: error: cannot find -lrosidl_generator_java__rosidl_typesupport_opensplice_c
/opt/ndk/android-ndk-r21d/toolchains/llvm/prebuilt/linux-x86_64/lib/gcc/arm-linux-androideabi/4.9.x/../../../../arm-linux-androideabi/bin/ld: error: cannot find -lrosidl_generator_java__rosidl_typesupport_opensplice_c
clang++: error: linker command failed with exit code 1 (use -v to see invocation)
make[2]: *** [rosidl_generator_java/rosidl_generator_java/srv/librosidl_generator_java_srv__basic_types__response__jni__rosidl_typesupport_opensplice_c.so] Error 1
make[1]: *** [CMakeFiles/srvBasicTypes_Responseopensplice_c.dir/all] Error 2
clang++: error: linker command failed with exit code 1 (use -v to see invocation)
make[2]: *** [rosidl_generator_java/rosidl_generator_java/msg/librosidl_generator_java_msg_w_strings__jni__rosidl_typesupport_opensplice_c.so] Error 1
make[1]: *** [CMakeFiles/msgWStringsopensplice_c.dir/all] Error 2
make: *** [all] Error 2


Failed   <<< rosidl_generator_java [0.65s, exited with code 2]

--- a/ros2_java_android.repos
+++ b/ros2_java_android.repos
@@ -99,10 +99,6 @@ repositories:
     type: git
     url: https://github.com/ros2/rmw_implementation.git
     version: dashing
-  ros2/rmw_opensplice:
-    type: git
-    url: https://github.com/ros2/rmw_opensplice.git
-    version: dashing
   ros2/rosidl:
     type: git
     url: https://github.com/ros2/rosidl.git
@@ -126,10 +122,6 @@ repositories:
   ros2/rosidl_typesupport_fastrtps:
     type: git
     url: https://github.com/ros2/rosidl_typesupport_fastrtps.git
-    version: dashing
-  ros2/rosidl_typesupport_opensplice:
-    type: git
-    url: https://github.com/ros2/rosidl_typesupport_opensplice.git
     version: dashing
   ros2/test_interface_files:
     type: git

